src/app/pages/DashboardPage/ParaTimeSnapshot.tsx

```diff
@@ -16,6 +16,7 @@ import { useConstant } from '../../hooks/useConstant'
 import { AppendMobileSearch } from '../../components/AppendMobileSearch'
 import { useRequiredScopeParam } from '../../hooks/useScopeParam'
 import { Network } from '../../../types/network'
+import { getLayerNames } from '../../../types/layers'
```

Comment:
at some point we could move this helper function out of `types` folder

Reply:
Agreed. Let's do that after the currentl quick series of PRs is finished.